src/workloads/scale/MixedWorkloadsGennyRateLimited.yml

```diff
@@ -1,14 +1,13 @@
 SchemaVersion: 2018-07-01
 Owner: "@mongodb/product-perf"
 Description: |
-  This workload is a port of the mixed_workloads in the workloads
-  repo. https://github.com/10gen/workloads/blob/master/workloads/mix.js. It runs 4 sets of
-  operations, each with dedicated actors/threads. The 4 operations are insertOne, findOne,
-  updateOne, and deleteOne. Since each type of operation runs in a dedicated thread it enables
-  interesting behavior, such as reads getting faster because of a write regression, or reads being
-  starved by writes. The origin of the test was as a reproduction for BF-2385 in which reads were
-  starved out by writes.
-
+  Phase defintions for the MixedWorkloadsGennyRateLimited, which is a port of the mixed_workloads in the
```

Comment:
This is the workload, not the phases, yes?

Reply:
Yes, this was in the previous file so I thought I should leave it in?